infinispan-remote/src/test/java/org/hibernate/ogm/datastore/infinispanremote/utils/InfinispanRemoteJpaServerRunner.java

```diff
@@ -32,6 +32,7 @@ public InfinispanRemoteJpaServerRunner(Class<?> klass) throws InitializationErro
 	@Override
 	public void run(RunNotifier notifier) {
 		try {
+			InfinispanRemoteTestHelper.deployJavaStoredProcedures();
```

Comment:
@fax4ever , I moved the method call here. WDYT?

Reply:
nice!